atom/browser/autofill/atom_autofill_client.cc

```diff
@@ -245,6 +245,8 @@ void AtomAutofillClient::DidFillOrPreviewField(
     const base::string16& profile_full_name) {
 }
 
+void AtomAutofillClient::DidInteractWithNonsecureCreditCardInput() {}
```

Comment:
cc @darkdh

Reply:
seem like we should call `InsecureSensitiveInputDriverFactory::
    DidInteractWithNonsecureCreditCardInput` or at least do something similar